src/en/spyfakku/src/eu/kanade/tachiyomi/extension/en/spyfakku/SpyFakku.kt

```diff
@@ -45,28 +53,77 @@ class SpyFakku : HttpSource() {
         .set("Origin", baseUrl)
 
     override fun popularMangaRequest(page: Int): Request {
-        return GET("$baseApiUrl/library?sort=released_at&page=$page", headers)
+        return GET("$baseUrl$baseApiEnd?sort=released_at&page=$page", headers)
     }
 
     override fun popularMangaParse(response: Response): MangasPage {
         val library = response.parseAs<HentaiLib>()
+        val data = getMangasDatas(library.data)
```

Comment:
![image](https://github.com/user-attachments/assets/018d8255-d5e4-4177-97c1-fda46ef26ad8)
response is like this which doesn't seem to match your parsing, maybe they changed it? or am I missing something?

Reply:
I added .substringAfter("\n") in Response.parseAs() to get only line 2's object. Their source seem to do the same.